Tools/LyTestTools/ly_test_tools/o3de/editor_test.py

```diff
@@ -0,0 +1,751 @@
+# Copyright (c) Contributors to the Open 3D Engine Project
```

Comment:
I'm unsure if this file will be picked up by pytest runners because the file name ends in _test

Reply:
Good point, just checked and It doesn't:

```
D:\o3de\branch_automation> python\python.cmd -i -m pytest Tools/LyTestTools/ly_test_tools/o3de/ --build-directory build\bin\profile\ -v
================================================================================= test session starts =================================================================================
platform win32 -- Python 3.7.10, pytest-5.3.2, py-1.9.0, pluggy-0.13.1 -- D:\o3de\branch_automation\python\runtime\python-3.7.10-rev1-windows\python\python.exe
cachedir: .pytest_cache
rootdir: D:\o3de\branch_automation
plugins: mock-2.0.0, timeout-1.3.4, ly-test-tools-1.0.0
collected 0 items
```

Reply:
Thats a good point, there are no tests defined here, so that wouldn't be a problem right now:
```
PS D:\o3de\branch_automation> python\python.cmd -i -m pytest Tools/LyTestTools/ly_test_tools/o3de/ --build-directory build\bin\profile\ -v
================================================================================= test session starts =================================================================================
platform win32 -- Python 3.7.10, pytest-5.3.2, py-1.9.0, pluggy-0.13.1 -- D:\o3de\branch_automation\python\runtime\python-3.7.10-rev1-windows\python\python.exe
cachedir: .pytest_cache
rootdir: D:\o3de\branch_automation
plugins: mock-2.0.0, timeout-1.3.4, ly-test-tools-1.0.0
collected 0 items
```

However you are right that this file will be picked. I tried adding a test_ function and it collected it.
I have updated the code to add __test__ = False which will make this file to never be picked by it, so if in the future something with test_ is created in this file, it never gets picked by the runner